activiti-core/activiti-engine/src/main/java/org/activiti/engine/impl/util/ProcessDefinitionUtil.java

```diff
@@ -54,20 +55,38 @@ public static ProcessDefinition getProcessDefinition(String processDefinitionId,
     }
   }
 
+  public static ProcessDefinitionHelper getProcessDefinitionHelper() {
+      ProcessDefinitionHelper processDefinitionHelper = Context.getProcessDefinitionHelper();
+
+        if (processDefinitionHelper == null) {
+            throw new ActivitiException("Found Old Activiti process definition, but no compatibility handler on the classpath");
```

Comment:
Update exception message: `No ProcessDefinitionHelper set in the current context`